File: C/Vetores/Media_pares.c
#include <stdio.h>
#include <string.h>


int main ()
{

    int N, i, soma, cont;
    double media;

    printf ("Quantos elementos vai ter o vetor? ");
    scanf ("%d", &N);

    int vet[N];

    for (i = 0; i < N; i++)
    {
        printf ("Digite um numero: ");
        scanf ("%d", &vet[i]);
    }

    cont = 0;
    soma = 0;

    for (i = 0; i < N; i++)
    {
        if (vet[i] % 2 == 0)
        {
            cont = cont + 1;
            soma = soma + vet[i];
        }
    }

        if (cont != 0){
        media =  (double) soma / cont;
        printf ("MEDIA DOS PARES: %.1lf\n", media);
        }
        else {
            printf ("NENHUM NUMERO PAR\n");
        }


    return 0;
}

File: Java/Vetores/Media_pares.java
package Java.Vetores;

import java.util.Locale;
import java.util.Scanner;


public class Media_pares {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);


        int N, i;
        
        
        System.out.print("Quantos elementos vai ter 0 vetor? ");
        N = sc.nextInt();

        int[] vet = new int[N];


        for (i = 0; i < N; i++) {
        	System.out.print("Digite um numero: ");
        	vet[i] = sc.nextInt();
        }
        
        int cont = 0;
        int soma = 0;
        double media = 0;
        
        for (i = 0; i < N; i++) {
        	if (vet[i] % 2 == 0) {
        		cont++;
        		soma = soma + vet[i];
        	}	
        }
        
        	if (cont == 0) {
        		System.out.println("NENHUM NUMERO PAR");
        	}
        	else {
        		 media = (double) soma / cont;
        		System.out.println("MEDIA DOS PARES = " + String.format("%.1f", media));
        	}
        
        sc.close();

    }
}
